package.json

```diff
@@ -16,6 +16,7 @@
   },
   "dependencies": {
     "@apollo/client": "^3.7.9",
+    "@apollo/server": "^4.11.0",
```

Comment:
Do we need apollo server?
```suggestion
```

Reply:
I encountered an error saying that the app could not resolve @apollo/server when I began the exam. Installing it fixed the error

Reply:
@eduardconstantin @apollo/server package is being used [here](https://github.com/eduardconstantin/azure-cloud-exams/blob/main/app/api/graphql/route.ts) in line 3

Reply:
That's strange, the deployed app is working fine without it